examples/AMY_MIDI_Synth/AMY_MIDI_Synth.ino

```diff
@@ -50,9 +44,6 @@ void setup() {
   amy_config.has_reverb = 1;
   amy_config.has_echo = 1;
 
-  // If you want the default synths (Juno on channel 1, drums on channel 10)
-  amy_config.set_default_synth = 1;
-  amy_config.startup_bleep = 1;
```

Comment:
I don't know (offhand) what the defaults are (except I think startup_bleep defaults to off).  My point here is that setting them, even if redundant with the defaults, is helpful to the reader/user for (a) being explicit and (b) providing awareness.  To do this with a bit mask is less readable, i.e.
```
amy_config.features |= AMY_FEATURE_DUALCORE;
amy_config.features &= ~AMY_FEATURE_AUDIO_IN;
...
```

Reply:
fixed with bitfields